cmd/boulder-va/main.go

```diff
@@ -132,11 +136,24 @@ func main() {
 		resolver = r
 	}
 
+	var remotes []va.RemoteVA
+	if len(c.VA.RemoteVAs) > 0 {
+		tls, err := c.VA.TLS.Load()
```

Comment:
Why load this once per loop? This should already be called somewhere else in main, just hoist it up to where it's usable for this?